e2e-tests/global-setup.js

```diff
@@ -42,7 +51,10 @@ const authenticateToBoundary = async () => {
 
 export default globalSetup;
 
-export const authenticatedState = './admin/artifacts/authenticated-state.json';
+export const authenticatedState = path.resolve(
```

Comment:
I moved the authentication file to its own `.auth` folder, as it seems [suggested](https://playwright.dev/docs/auth#core-concepts) by playwright